lib/element/dotnet/dotnetCoreScanner.ts

```diff
@@ -61,6 +63,7 @@ export const dotnetCoreScanner: TechnologyScanner<DotnetCoreStack> = async p =>
         const stack: DotnetCoreStack = {
             tags: ["dotnetcore"],
             name: "dotnetcore",
+            hasDockerFile: await HasDockerfile.predicate(p),
```

Comment:
This isn't searching the entire project structure. Only the root `Dockerfile` please change to something like:

```typescript
let dockerfile: string = "Dockerfile";
await projectUtils.doWithFiles(p, "**/Dockerfile", async f => {
    dockerfile = f.path;
});
```

^ that should probably be a function by itself.

Reply:
Perhaps we also need to change that pushtest?